features/step_definitions/common_steps.rb

```diff
@@ -322,16 +320,12 @@
 end
 
 Then /^I see '(.*)' in the '(.*)' summary table$/ do |content, table_heading|
-  result_table_location = "//*[@class='summary-item-heading'][normalize-space(text())=\"#{table_heading}\"]/following-sibling::*[1]"
-  result_table_rows_location = result_table_location + "/tbody/tr[@class='summary-item-row']"
-  result_table_rows = all(:xpath, result_table_rows_location)
+  result_table_rows = get_table_rows_by_caption(table_heading)
   result_table_rows.any? {|row| row.text.include? content}.should be true
 end
 
 Then /^I see that the '(.*)' summary table has (\d+)(?: or (more|fewer))? entr(?:y|ies)$/ do |table_heading, expected_number_of_rows, comparison|
-  result_table_location = "//*[@class='summary-item-heading'][normalize-space(text())=\"#{table_heading}\"]/following-sibling::*[1]"
-  result_table_rows_location = result_table_location + "/tbody/tr[@class='summary-item-row']"
-  number_of_table_rows = all(:xpath, result_table_rows_location).length
+  number_of_table_rows = get_table_rows_by_captiong(table_heading).length
```

Comment:
I think there might be a typo here: "get_table_rows_by_captiong" instead of "get_table_rows_by_caption"

Reply:
Good catch, thanks! Updated